examples/views/button/paginator.py

```diff
@@ -16,39 +16,46 @@ def __init__(self, embeds: List[disnake.Embed]):
         # Current embed number.
         self.embed_count = 0
 
+        # Disables previous page button by default.
+        self.last_page.disabled = True
```

Comment:
I thought there was a bug with doing this that hasn't been resolved yet?

Reply:
Do you mean this PR #163?